### DISCUSSION TITLE
Request removal of ALVR.ALVR package

### DISCUSSION THREAD

**Discussion by zmerp** [microsoft/winget-pkgs#125124] [General]
Hello, I'm the lead developer of ALVR. The package ALVR.ALVR was submitted without our knowledge, and I would like to request the removal from the winget repository. The reason is, ALVR updates frequently and more often than not the winget package is out of date, and this creates confusion for the end user. We have other download sources we maintain and we don't plan to maintain the winget package.

**Comment by mdanish-kh** [comment]
PRs raised:

- #125189
- #125190
- #125196

**Reply by zmerp** [reply]
Thanks!
